sunbird-mentoring/index.js

```diff
@@ -40,10 +41,52 @@ const createPackage = (options) => {
 	}
 }
 
+const packageName = 'kb-mentoring-notification'
```

Comment:
@rakeshSgr can we change this to sunbird notification?

Reply:
if we change this, we need to change the env's in KB.